lib/mixins/element-mixin.js

```diff
@@ -760,7 +749,7 @@ function _regLog(prototype) {
  */
 export function register(prototype) {
   registrations.push(prototype);
-  undefined && _regLog(prototype);
```

Comment:
I don't think this is right. In 2.x this only logged if `Polymer.log` was set. Modulizer made it never log, which isn't great, but after this change we'll always log, which we also don't want. 

IMO we should invent a global name that, when truthy, means that we'll log here.

Reply:
@rictic Ah crap, you are correct! Reinvent the `Polymer.log` setting? 😂

Reply:
I'd say let's just remove the registration logging altogether in 3.x. I don't know that this feature was ever [documented](https://www.polymer-project.org/2.0/docs/devguide/settings), and the whole registration tracking system is fairly dubious now that WC V1 has a registry API.

Will make a new PR to remove that line; `dumpRegistrations` can remain.

Reply:
https://github.com/Polymer/polymer/pull/5331